Makefile

```diff
@@ -22,6 +22,24 @@ CALICO_CNI_PLUGIN_BUILD = 1
 ALL = \
 	$(ISO_ROOT)/bootstrap.sh \
 	\
+	$(ISO_ROOT)/salt/metalk8s/orchestrate/bootstrap_step1.sls \
```

Comment:
Could be interesting to know the difference between `step1` and `step2` just by reading their names.
I think `step1` is `before_salt_master` and `step2` is `after...`?